unitary/alpha/quantum_world.py

```diff
@@ -484,6 +486,35 @@ def get_histogram(
                 histogram[idx][cast(int, result[idx])] += 1
         return histogram
 
+    def get_correlated_histogram(
+        self, objects: Optional[Sequence[QuantumObject]] = None, count: int = 100
+    ) -> Dict[Tuple[int], int]:
+        """Creates histogram of the whole quantum world (or `objects` if specified)
+        based on measurements (peeks) carried out. Comparing to get_histogram(),
+        this statistics contains entanglement information accross quantum objects.
+
+        Parameters:
+            objects:    List of QuantumObjects
+            count:      Number of measurements
+
+        Returns:
+            A dictionary, with the keys being each possible state of the whole quantum
+            world (or `objects` if specified) in terms of tuple, and the values being
```

Comment:
Let's clarify that the key is a tuple of the results of each object in `objects` and in the order of that argument.

Reply:
Thanks Doug. It's updated.